musicAlbum/musicalbum.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(on_spotify, publish_date)
+    super(publish_date)
+    @on_spotify = on_spotify
+  end
```

Comment:
- [ ] Nice Job on adding the requirements, however, the music album is missing the `label, and source, and Author` for the `music`, and as I see in the **kanban** please add them since it's the requirement for 4 people project.

![Capture web_1-7-2022_122952_gitlab com](https://user-images.githubusercontent.com/61562626/176886393-0be7e767-e1e7-4659-ace3-db00692d9d72.jpeg)

Reply:
## Hi @dasileker 
Thanks for the feedback. I just have one question. Since the MucisAlbum class inherits from the `Item` class all these properties of `Item` class are also inherited. Do we need to write them in the constructor again?